.github/workflows/prerelease.yml

```diff
@@ -32,11 +32,6 @@ jobs:
       - run: ./build.sh release -c
         name: Release
         
-      - uses: actions/upload-artifact@v4
```

Comment:
@v1v I don't remember. Are we uploading the artifact for a specific reason?

Reply:
I believe it was related to the provenance step running in `L40`, otherwise, it will not be possible to validate those artifacts.

Reply:
This is a reversal of #135 that I added